addons/l10n_in_ewaybill/tools/ewaybill_api.py

```diff
@@ -48,11 +48,11 @@ def get_error_codes(self):
 
 class EWayBillApi:
 
-    DEFAULT_HELP_MESSAGE = _lt(
+    DEFAULT_HELP_MESSAGE = str(_lt(
```

Comment:
It is not a good idea to have this with str, because if the translation does not have %s, it will fail harshly.  Why not change it into a lambda method?  
```suggestion
    DEFAULT_HELP_MESSAGE = lambda l: _lt("Somehow %s bla", l)
```

But the weird thing is that "generated" is not translated at all and I hope there are no languages where generated splits in two words separated from each other.

Reply:
The fix should be somewhat @jco-odoo suggested. But don't even use lazy translations (many people misuse them).
The `%` on the left is supposed to be translated, but not the injected status value.

```python
def default_help_message(self, status):
  return self.env._("""Something %s""", status)

# call with
self.default_help_message(self.env._("cancelled"))
```